lib/archethic/p2p/message.ex

```diff
@@ -1716,10 +1716,22 @@ defmodule Archethic.P2P.Message do
   def process(%GetBeaconSummariesAggregate{date: date}) do
     case BeaconChain.get_summaries_aggregate(date) do
       {:ok, aggregate} ->
-        aggregate
+        %BeaconSummariesAggregate{aggregate: aggregate}
 
       {:error, :not_exists} ->
         %NotFound{}
     end
   end
+
+  def process(%BeaconSummariesAggregate{aggregate: aggregate}) do
```

Comment:
I don't think you need to use a specific message if you are doing pull mechanism